addons/mail/static/src/utils/common/misc.js

```diff
@@ -191,3 +192,31 @@ export function convertToEmbedURL(url) {
     }
     return { url: null, provider: null };
 }
+
+export async function loadBundleInShadow(shadow, bundleName) {
```

Comment:
Is this function actually specific to a shadow what so ever?

The only use is for `appendChild` so it could work on any `Node`?

Reply:
I renamed to `loadCssFromBundle` for now.